README.md

```diff
@@ -0,0 +1,38 @@
+# Roulette Telegram Chat
+
+![Man speaking with women](./docs/logo.png)
+The idea is to create a chat for communication between two random people without revealing their identities.
+
+## Use case diagram
+
+![Interaction between user, moderator and system](./docs/useCase.png)
+Main points:
+
+### User
+
+- can register in system(by pressing button start)
+- can change companion to new one
+- can stop communication
+- report companion abuse
+- block companion to not interact with him anymore
+- get companion "fake" identity (name, age, hobbies)
+- watch own identity
+- edit own identity
+- set random identity (default action on start of the bot)
+
+### Moderator
+
+- can ban user (for inappropriate behaviour(Look more in "inapropriate behavior"))
+- watch logs by user
```

Comment:
what data do we store?
do we index that store?

Reply:
I will create ER diagram

Reply:
ok